src/utils/const.ts

```diff
@@ -15,15 +15,36 @@
  * limitations under the License.
  */
 
-export const VERSION_DETAIL = "0.1.58"
+import { requestAPI } from '../handler/handler';
+export const VERSION_DETAIL = "0.1.66"
 export const CREATE_CLUSTER_URL =
   'https://console.cloud.google.com/dataproc/clusters';
 export const CREATE_BATCH_URL =
   'https://console.cloud.google.com/dataproc/batches/create';
-export const BASE_URL = 'https://dataproc.googleapis.com/v1';
-export const BASE_URL_NETWORKS = 'https://compute.googleapis.com/compute/v1';
-export const BASE_URL_META = 'https://metastore.googleapis.com/v1';
-export const BASE_URL_KEY = 'https://cloudkms.googleapis.com/v1';
+
+interface IGcpUrlResponseData {
+  dataproc_url: string;
+  compute_url: string;
+  metastore_url: string;
+  cloudkms_url: string;
+  cloudresourcemanger_url: string;
+  datacatalog_url: string;
+  storage_url: string;
+}
+
+const data = await requestAPI('getGcpServiceUrls');
```

Comment:
Can we not do this in an await?  This is a blocking call while the javascript code initializes.

Reply:
@ptwng we tried removing that, but looks like we are not able to achieve what we are looking for. Please let us know if we need to try something else. The const.ts file is just static file with no react components. We do not see any lag in the UI as of now.